server/http.go

```diff
@@ -434,7 +435,7 @@ func handleTokenFunc(srv OIDCServer) http.HandlerFunc {
 			return
 		}
 
-		creds := oidc.ClientCredentials{ID: user, Secret: password}
+		creds := oidc.ClientCredentials{ID: user, Secret: base64.URLEncoding.EncodeToString([]byte(password))}
```

Comment:
I'm a little confused what the intention of this change is. The issues with the --no-db mode secrets required to be base64 encoded are in the db package.